pam/pam_adsys.c

```diff
@@ -223,7 +224,18 @@ static char *get_default_sss_domain(pam_handle_t *pamh) {
                 pam_syslog(pamh, LOG_ERR, "Could not find value for key 'default_domain_suffix' in sssd.conf");
                 break;
             }
-            while (*++domain == ' ') continue;
+            // Ignores whitespaces and tabs right after the '='
+            do {
+                domain++;
+            } while (strlen(domain) > 0 && (*domain == ' ' || *domain == '\t'));
```

Comment:
Is it better, though?

Reply:
There isn't a one liner, I think. The shortest way would be:
```C
do domain++;
while (strlen(domain) > 0 && (*domain == ' ' || *domain == '\t'));
```
But it looks so wrong